specs/protocol/revenue-sharing.md

```diff
@@ -0,0 +1,68 @@
+<!-- START doctoc generated TOC please keep comment here to allow auto update -->
+<!-- DON'T EDIT THIS SECTION, INSTEAD RE-RUN doctoc TO UPDATE -->
+**Table of Contents**
+
+- [Revenue Sharing](#revenue-sharing)
+  - [Definitions](#definitions)
+  - [`RevenueSharer` predeploy](#revenuesharer-predeploy)
+    - [Deploying `RevenueSharer`](#deploying-revenuesharer)
+      - [Existing Chains](#existing-chains)
+      - [Chains after version X.Y.Z of OP Stack](#chains-after-version-xyz-of-op-stack)
+    - [Execution](#execution)
+  - [Simplified L1 Data Fee Expenditure](#simplified-l1-data-fee-expenditure)
+
+<!-- END doctoc generated TOC please keep comment here to allow auto update -->
+
+# Revenue Sharing
```

Comment:
Do you mind moving `# Revenue Sharing` to the top of this doc and then rename this line to `Overview`? This will match the other docs

Reply:
[952383f](https://github.com/ethereum-optimism/specs/pull/115/commits/952383fa75386a2599f734e95fcd45a1ea05dbdc)